samples/nrf9160/modem_shell/src/uart/uart_shell.c

```diff
@@ -99,13 +99,13 @@ void uart_toggle_power_state(void)
 			/* allow little time for printing the notification */
 			k_sleep(K_MSEC(500));
 
-			/* set uart0 to low power state */
-			err = pm_device_state_set(uart_dev, PM_DEVICE_STATE_LOW_POWER);
```

Comment:
LGTM if this does the required result (=same level pwr consumption than earlier when uarts disabled), but was it clarified that where/why did the PM_DEVICE_STATE_LOW_POWER disappeared from used serial driver?

Reply:
Supposedly the driver was "simplified". Actually even before, requesting low power state really caused only a fallthrough to PM_DEVICE_STATE_OFF, see https://github.com/nrfconnect/sdk-zephyr/commit/9e8e88aee4b5d66002b0bf7373ca56b935aecf18

Reply:
As background, when I first implemented disabling of the UARTs to modem_shell, the code was changed during the review from setting registers directly to use the Zephyr Device Power Management API. The state suggested in the review was PM_DEVICE_STATE_LOW_POWER and because that worked fine, that one was left in the code. That's the reason why PM_DEVICE_STATE_LOW_POWER was used in the first place.